src/main/java/io/kestra/plugin/notifications/sentry/SentryExecution.java

```diff
@@ -34,7 +34,6 @@ The alert message will include a link to the execution page in the UI along with
                 tasks:
                   - id: send_alert
                     type: io.kestra.plugin.notifications.sentry.SentryExecution
-                    executionId: "{{ trigger.executionId }}"
```

Comment:
Why do you remove this line?

Reply:
Because the line was duplicated in Line 41

Reply:
OK, good catch